client/package.json

```diff
@@ -62,6 +63,7 @@
     ]
   },
   "devDependencies": {
+    "@types/audiosprite": "^0",
```

Comment:
Unless we are using audiosprite somewhere other than the terminal, there is no need for typings